public/css/widget/quick-actions.less

```diff
@@ -2,27 +2,35 @@
   display: flex;
   flex-wrap: wrap;
   list-style-type: none;
-  // TODO: Eh? Minus .25em on the left and right, maybe, because padding and responsiveness.
-  //       But what is the additional 1em on the left for???
-  margin: 0 -1.25em 0 -.25em;
-  margin: 0;
+  margin: 0 -.5em;
```

Comment:
What does the decrease by .25em accomplish in this regard? It's horizontal, why is it necessary for the controls alignment?

Reply:
I'll leave this as discussed.